<ui version="4.0" >
 <class>CurvePlacement</class>
 <widget class="QWidget" name="CurvePlacement" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>605</width>
    <height>262</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>0</number>
   </property>
   <item row="0" column="0" >
    <widget class="QGroupBox" name="_curvePlacementGroup" >
     <property name="title" >
      <string>Placement</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" colspan="2" >
       <widget class="QRadioButton" name="_noPlot" >
        <property name="text" >
         <string>Do not place in any plot</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QRadioButton" name="_existingPlot" >
        <property name="text" >
         <string>Place in &amp;existing plot:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QComboBox" name="_plotList" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="whatsThis" >
         <string>The name of the data file which will be read for this vector.</string>
        </property>
        <property name="editable" >
         <bool>false</bool>
        </property>
        <property name="insertPolicy" >
         <enum>QComboBox::InsertAtTop</enum>
        </property>
        <property name="autoCompletion" >
         <bool>true</bool>
        </property>
        <property name="duplicatesEnabled" >
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QRadioButton" name="_newPlot" >
        <property name="text" >
         <string>Place in &amp;new plot</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="1" >
       <widget class="QGroupBox" name="_layoutGroup" >
        <property name="title" >
         <string/>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" colspan="2" >
          <widget class="QRadioButton" name="_autoLayout" >
           <property name="text" >
            <string>Automatic Layout</string>
           </property>
           <property name="checked" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0" >
          <widget class="QRadioButton" name="_customGrid" >
           <property name="text" >
            <string>Custom Grid</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="label_2" >
             <property name="text" >
              <string>Columns:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="_gridColumns" >
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2" >
          <widget class="QRadioButton" name="_protectLayout" >
           <property name="text" >
            <string>Protect Existing Layout</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>_newPlot</tabstop>
  <tabstop>_plotList</tabstop>
  <tabstop>_existingPlot</tabstop>
  <tabstop>_noPlot</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
